### PR TITLE
Include the strings for the next desktop promo update

### PR DESCRIPTION
**Pull request by mstriemer**
This just includes the strings for the next desktop promo update in an unused array so that they can be extracted early.

### CODE REVIEW

File: src/media/js/desktop_promo.js [hunk 1]
```diff
@@ -8,6 +8,24 @@ define('desktop_promo',
     'use strict';
     var gettext = l10n.gettext;
 
+    var nextPromoItems = [
```
Comment (ngokevin) [inline src/media/js/desktop_promo.js]: I'd add a comment

**Comment by mstriemer** [comment]
Not much to see here. Going to merge it.

**Comment by ngokevin** [comment]
but yeah r+

File: src/media/js/desktop_promo.js (hunk 1 above)
Comment (mstriemer) [inline src/media/js/desktop_promo.js]: Done.

**Pull request merged**